File: python/prism_sdk/throughput_research_bundle_integrity_inference.py
"""Bundle P32 throughput inference signed research-object integrity feature."""
from .research_bundle_integrity_support import BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError,manifest,release
FEATURE_ID="AFA-bundle-P32-F03";CONTRACT_VERSION="bundle-throughput_research_bundle_integrity_inference/1.0"
def throughput_research_bundle_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
def release_throughput_research_bundle_integrity_inference(request:BundleReleaseRequest4)->BundleCard7:return release(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="throughput",mode="inference")
__all__=["FEATURE_ID","CONTRACT_VERSION","throughput_research_bundle_integrity_inference_manifest","release_throughput_research_bundle_integrity_inference"]

File: crates/bundle/src/throughput_research_bundle_integrity_inference.rs
//! Bundle P32 throughput inference signed research-object integrity feature.
use super::research_bundle_integrity_support::{manifest,release,BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError};
pub const FEATURE_ID:&str="AFA-bundle-P32-F03";pub const CONTRACT_VERSION:&str="bundle-throughput_research_bundle_integrity_inference/1.0";
pub fn throughput_research_bundle_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
pub fn release_throughput_research_bundle_integrity_inference(request:&BundleReleaseRequest4)->Result<BundleCard7,ResearchBundleIntegrityError>{release(request,FEATURE_ID,CONTRACT_VERSION,"throughput","inference")}
